### PR TITLE
Weekly updates for week 1,2,3,4 of Theming and configuration

### PR DESCRIPTION
**Pull request by Tushar98644**
## Project

Theming and configuration

### Changes

What was updated (new pages, weekly update, etc.)

### Notes

If there are any further notes.

@Apoorvgarg-creator Kindly verify

### CODE REVIEW

File: docs/2023/Sunbird UCI/Theming and configuration/updates/2023-07-07.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Contributions
+-  Setup the project locally, made branches for development 
+-  Setup config for production and development modes  
+-  Whenever feasible, localstorage has been substituted with environment varibles.
+
+## Learnings
+Got a walkthrough of the codebase and the architecture of the app by mentor, attended meetings for doubts. Learned how config is setup for a turborepo. 
+
+ #### Link to PR: https://github.com/samagra-comms/uci-web-channel/pull/130
```
Comment (shubhigupta991) [inline docs/2023/Sunbird UCI/Theming and configuration/updates/2023-07-07.md:22]: Attatch this link under contribution section

File: docs/2023/Sunbird UCI/Theming and configuration/updates/2023-07-14.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Contributions 
+- Removed redundant code and used index.ts file for exports for files and images thereby reducing the imports 
+- Added new files for passing config from the components  
+- Raised a PR for this week and last week's work 
+
+## Learnings
+ Learned how index.ts file works and how exports can be used , passing config for the components ,  
+
+ #### Link to PR: https://github.com/samagra-comms/uci-web-channel/pull/130
```
Comment (shubhigupta991) [inline docs/2023/Sunbird UCI/Theming and configuration/updates/2023-07-14.md:21]: same.

File: docs/2023/Sunbird UCI/Theming and configuration/updates/2023-07-21.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+![](https://i.postimg.cc/bJtsHN8p/Screenshot-2023-07-23-at-10-55-52-PM.png)
+
+## Contributions
+-  Refactored the entire look of the homepage, chatpage and starred-messages page 
+- Made a custom component for the chat window for better configuration
+
+## Learnings
+Recently delved into the concept of mono repos, like Turborepo, which promote modular and scalable projects. Additionally, I grasped the significance of implementing Redux store to enhance application scalability and modularity, ensuring efficient state management and maintainability.
+
+#### Link to PR: https://github.com/samagra-comms/uci-web-channel/pull/132
```
Comment (shubhigupta991) [inline docs/2023/Sunbird UCI/Theming and configuration/updates/2023-07-21.md:21]: same

**Review by shubhigupta991 — CHANGES_REQUESTED** [review]
_category_.json in docs/2023/Sunbird UCI/Theming and configuration folder

File: docs/2023/Sunbird UCI/Theming and configuration/index.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 1
+title: Introduction
+slug: /2023/Sunbird UCI/Tushar Banik
```
Comment (Apoorvgarg-creator) [inline docs/2023/Sunbird UCI/Theming and configuration/index.md:4]: Name must match the path of the file

File: docs/2023/Sunbird UCI/Theming and configuration/updates/2023-07-07.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+author: Tushar Banik   
+---
+
+## Milestones
+- [x] Setting up config for development and production modes
+
+## Screenshots / Videos 
+#### Config for urls
+![](https://i.postimg.cc/v8544B3v/Screenshot-2023-07-25-at-9-11-44-PM.png)
+![](https://i.postimg.cc/BnmLJVM9/Screenshot-2023-07-25-at-9-11-56-PM.png)
```
Comment (Apoorvgarg-creator) [inline docs/2023/Sunbird UCI/Theming and configuration/updates/2023-07-07.md:12]: Why are there https links ? 
And also appropriate name for files

**Review by Apoorvgarg-creator — CHANGES_REQUESTED** [review]

**Comment by GeekGawd** [comment]
@Tushar98644 Could you add alt text to it please

**Comment by GeekGawd** [comment]
@Tushar98644 add the ppt and the mentor feedback also kindly add the alt text to the images please.

**Comment by Tushar98644** [comment]
> @Tushar98644 add the ppt and the mentor feedback also kindly add the alt text to the images please.

Done

**Review by Apoorvgarg-creator — APPROVED** [review]

**Review by shubhigupta991 — APPROVED** [review]

**Pull request merged**